.schema/spicepod.schema.json

```diff
@@ -809,21 +701,15 @@
     },
     "SpicepodKind": {
       "type": "string",
-      "enum": [
-        "Spicepod"
-      ]
+      "enum": ["Spicepod"]
     },
     "SpicepodVersion": {
       "type": "string",
-      "enum": [
-        "v1beta1"
-      ]
+      "enum": ["v1beta1"]
     },
     "TelemetryConfig": {
       "type": "object",
-      "required": [
-        "enabled"
-      ],
+      "required": ["enabled"],
```

Comment:
Looks like the auto-formatter kicked in - can you redo the change without saving the formatting changes? I think the workflow will change this back.

Reply:
Yes, in fact the workflow will update this .json file, so it doesn't need to be manually here in this PR.